data/seeds/01-projects.js

```diff
@@ -0,0 +1,13 @@
+exports.seed = function(knex, Promise) {
+    return knex('projects').del()
```

Comment:
Don't use delete here unless you want ids to always increment. Use truncate instead.